Lib/ufo2ft/outlineCompiler.py

```diff
@@ -636,14 +636,18 @@ def _setupTable_hhea_or_vhea(self, tag):
                 continue
             if isHhea:
                 boundsAdvance = (bounds.xMax - bounds.xMin)
+                # equation from the hhea spec for calculating xMaxExtent:
+                #   Max(lsb + (xMax - xMin))
+                extent = firstSideBearing + boundsAdvance
             else:
                 boundsAdvance = (bounds.yMax - bounds.yMin)
+                # equation from the vhea spec for calculating yMaxExtent:
+                #   Max(tsb - (yMax - yMin)).
```

Comment:
oh I missed that. Was it correct before this PR?

Reply:
No, it may not have been in some cases: https://github.com/googlei18n/ufo2ft/pull/233/files#diff-2e916b2b1efc6fc438bed046c8c37f01L706
I’m looking into why the test hasn’t changed after the fixup commit.

Reply:
that "equation" in the OT spec is confusing...
the Apple TTRefMan has this definition for "extent"

> The 'hhea' table uses the concept of extent. The extent is the distance from the left side bearing to the right most positions in the glyph outline.

Reply:
Before this PR, it did follow what [Apple TTRefMan vhea](https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6vhea.html) says: 
> This is defined as the value of the minTopSideBearing field added to the result of the value of the yMin field subtracted from the value of the yMax field.

But the [OTspec vhea](https://docs.microsoft.com/en-us/typography/opentype/spec/vhea) says:
> Defined as yMaxExtent = max(tsb - (yMax - yMin)).

Reply:
Yes. Originally on the OpenType mailing list, I suggested correcting from:

`yMaxExtent = minTopSideBearing + (yMax - yMin)` 

to:

`yMaxExtent = max(tsb + (yMax - yMin))`

in September 2017. Then the OpenType spec 1.8.2 was changed to:

`yMaxExtent = max(tsb - (yMax - yMin))`

...but I've never noticed the minus sign until now.😫 I don't know it is intentional or not.

Reply:
Thanks, Cosimo!

Reply:
In the OT spec, it has already been corrected.

> In the erratum of 14 September 2017, a typo was introduced during the change, resulting in an incorrect calculation. The correct calculation should be yMaxExtent = max(tsb + (yMax-yMin)).

https://docs.microsoft.com/en-us/typography/opentype/spec/errata

Reply:
@mashabow Thank you for following up!
@moyogo could you apply the change?